frontend/src/summary.pug

```diff
@@ -0,0 +1,192 @@
+            i.fas.fa-circle(
+              v-bind:style="{ 'color' : contributionBarFileTypeColors[fileType] }"
+            )
+            span &nbsp; {{ fileType }} &nbsp;
+        template(v-else)
+          .summary-charts__fileType--breakdown__legend(
+            v-for="fileType in Object.keys(contributionBarFileTypeColors)")
+            i.fas.fa-circle(
+              v-bind:style="{ 'color' : contributionBarFileTypeColors[fileType] }"
+            )
+            span &nbsp; {{ fileType }} &nbsp;
+      .summary-chart(v-for="(user, i) in repo")
+        .summary-chart__title(v-if="!isMergeGroup")
+          .summary-chart__title--index {{ i+1 }}
+          .summary-chart__title--repo(v-if="filterGroupSelection === 'groupByNone'") {{ user.repoName }}
+          .summary-chart__title--author-repo(v-if="filterGroupSelection === 'groupByAuthors'") {{ user.repoName }}
+          .summary-chart__title--name(
+            v-if="filterGroupSelection !== 'groupByAuthors'",
+            v-bind:class="{ warn: user.name === '-' }"
+          ) {{ user.displayName }} ({{ user.name }})
+          .summary-chart__title--contribution.mini-font [{{ user.totalCommits }} lines]
+          a(
+            title="click to view author's code",
+            onclick="deactivateAllOverlays()",
+            v-on:click="openTabAuthorship(user, repo, i)"
+          )
+            i.icon-button.fas.fa-code
+          a(
+            v-bind:href="getRepoLink(repo[i])", target="_blank",
+            title="click to view author's repo"
+          )
+            i.icon-button.fab.fa-github
+          a(
+            title="click to view breakdown of commits",
+            onclick="deactivateAllOverlays()",
+            v-on:click="openTabZoom(user, filterSinceDate, filterUntilDate, repo, i)"
+          )
+            i.icon-button.fas.fa-list-ul
+          .summary-chart__title--percentile.mini-font(v-if="filterGroupSelection === 'groupByNone'") {{ getPercentile(i) }} %
+
+        .summary-chart__ramp(
+          onclick="viewClick(event)",
+          v-on:click="openTabZoomSubrange(user, repo, i)"
+        )
+          v_ramp(
+            v-bind:groupby="filterGroupSelection",
+            v-bind:user="user",
+            v-bind:tframe="filterTimeFrame",
+            v-bind:sdate="filterSinceDate",
+            v-bind:udate="filterUntilDate",
+            v-bind:avgsize="avgCommitSize",
+            v-bind:mergegroup="isMergeGroup")
+          .overlay
+
+        .summary-chart__contribution
+          template(v-if="filterBreakdown")
+            .summary-chart__contrib(
+              v-for="(widths, fileType) in getFileTypeContributionBars(user.fileTypeContribution)"
+            )
+              .summary-chart__contrib--bar(
+                v-for="width in widths",
+                v-bind:style="{ width: width + '%',\
+                              'background-color': contributionBarFileTypeColors[fileType] }",
```

Comment:
Weird line wrap/indentation starting from here. Can you check so that the original line wrap/indentation is not affected? Or the line wrap/indentation is configured according to the #987?

Reply:
fixed